### PR TITLE
more checks

### PR DESCRIPTION
**Pull request by adecaro**

### CODE REVIEW

File: fungible.mk [hunk 1]
```diff
@@ -4,7 +4,7 @@ integration-tests-dlog-fabric:
 
 .PHONY: integration-tests-dlog-fabric-t1
 integration-tests-dlog-fabric-t1:
-	cd ./integration/token/fungible/dlog; export FAB_BINS=$(FAB_BINS); ginkgo $(GINKGO_TEST_OPTS) --focus "Fungible" .
+	cd ./integration/token/fungible/dlog; export FAB_BINS=$(FAB_BINS); ginkgo $(GINKGO_TEST_OPTS) --focus "Fungible with Auditor ne Issuer" .
```
Comment (HagarMeir) [inline fungible.mk:7]: Auditor and Issuer?

File: integration/token/fungible/dlog/dlog_test.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ var _ = Describe("EndToEnd", func() {
 		network.Stop()
 	})
 
-	Describe("Fungible", func() {
+	Describe("Fungible with Auditor ne Issuer", func() {
```
Comment (HagarMeir) [inline integration/token/fungible/dlog/dlog_test.go:27]: Auditor and Issuer?

File: fungible.mk (hunk 1 above)
Comment (adecaro) [reply fungible.mk:7]: I have used `ne` as `not equal`.

File: integration/token/fungible/dlog/dlog_test.go (hunk 2 above)
Comment (adecaro) [reply integration/token/fungible/dlog/dlog_test.go:27]: I have used `ne` as `not equal`.

File: integration/token/fungible/views/checks.go [hunk 3]
```diff
@@ -135,6 +139,48 @@ func (m *CheckTTXDBView) Call(context view.Context) (interface{}, error) {
 		}
 	}
 
+	// Match unspent tokens with the ledger
+
```
Comment (HagarMeir) [inline integration/token/fungible/views/checks.go:143]: Remove extra line

File: integration/token/fungible/views/checks.go [hunk 4]
```diff
@@ -135,6 +139,48 @@ func (m *CheckTTXDBView) Call(context view.Context) (interface{}, error) {
...
+	// but first delete the claimed tokens
+	// TODO: check all owner wallets
```
Comment (HagarMeir) [inline integration/token/fungible/views/checks.go:145]: Do you wish to keep this TODO?

File: token/services/network/fabric/network.go [hunk 5]
```diff
@@ -14,6 +14,8 @@ import (
 	"sync"
 	"time"
 
+	token2 "github.com/hyperledger-labs/fabric-token-sdk/token"
+
```
Comment (HagarMeir) [inline token/services/network/fabric/network.go:18]: please remove extra line

File: token/services/network/processor/common.go [hunk 6]
```diff
@@ -34,7 +34,7 @@ type RWSet interface {
 type TokenStore interface {
 	// DeleteFabToken adds to the passed rws the deletion of the passed token
 	// TODO: we should delete also the extra tokens for the ids
```
Comment (HagarMeir) [inline token/services/network/processor/common.go:36]: What about this TODO?

File: token/services/network/fabric/vault.go [hunk 7]
```diff
@@ -57,7 +59,7 @@ func (v *Vault) DeleteTokens(ns string, ids ...*token.ID) error {
 
 	wrappedRWS := &rwsWrapper{RWSet: rws}
 	for _, id := range ids {
-		if err := v.tokenStore.DeleteFabToken(ns, id.TxId, id.Index, wrappedRWS); err != nil {
+		if err := v.tokenStore.DeleteFabToken(ns, id.TxId, id.Index, wrappedRWS, string(debug.Stack())); err != nil {
```
Comment (HagarMeir) [inline token/services/network/fabric/vault.go:62]: Why is the `string(debug.Stack()` passed here?

File: token/services/network/orion/querytokens.go [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+		if err != nil {
+			errs = append(errs, errors.Errorf("error creating output ID: %s", err))
+			continue
+			// return nil, errors.Errorf("error creating output ID: %s", err)
+		}
+		logger.Debugf("query state [%s:%s]", id, outputID)
+		bytes, err := qe.Get(orionKey(outputID))
+		if err != nil {
+			errs = append(errs, errors.Wrapf(err, "failed getting output for [%s]", outputID))
+			// return nil, errors.Wrapf(err, "failed getting output for [%s]", outputID)
```
Comment (HagarMeir) [inline token/services/network/orion/querytokens.go:125]: do you wish to leave these comments here?

File: token/services/network/orion/vault.go [hunk 9]
```diff
@@ -54,7 +56,7 @@ func (v *Vault) DeleteTokens(ns string, ids ...*token.ID) error {
 
 	wrappedRWS := &rwsWrapper{RWSet: rws}
 	for _, id := range ids {
-		if err := v.tokenStore.DeleteFabToken(ns, id.TxId, id.Index, wrappedRWS); err != nil {
+		if err := v.tokenStore.DeleteFabToken(ns, id.TxId, id.Index, wrappedRWS, string(debug.Stack())); err != nil {
```
Comment (HagarMeir) [inline token/services/network/orion/vault.go:59]: Why is `string(debug.Stack()` passed here?

File: integration/token/fungible/views/checks.go (hunk 4 above)
Comment (adecaro) [reply integration/token/fungible/views/checks.go:145]: yes, I will address in another PR

File: integration/token/fungible/views/checks.go (hunk 3 above)
Comment (adecaro) [reply integration/token/fungible/views/checks.go:143]: ok

File: token/services/network/orion/querytokens.go (hunk 8 above)
Comment (adecaro) [reply token/services/network/orion/querytokens.go:125]: okay

File: token/services/network/processor/common.go (hunk 6 above)
Comment (adecaro) [reply token/services/network/processor/common.go:36]: I'll remove, this was already addressed

**Review by HagarMeir — APPROVED** [review]

**Pull request merged**